coordinator/src/node.rs

```diff
@@ -259,17 +259,16 @@ fn build_contract_descriptor(
 /// function like we used to do in ItchySats.
 fn build_payout_curve(
     total_collateral: u64,
-    execution_price: f64,
+    initial_price: f64,
```

Comment:
🤔 I don't think `initial_price` is completely right here, as this function will also be used with the `close_price`. I found `execution_price` to fitting best to reflect both kind of prices.

Note, I agree on the `build_contract_descriptor` function

Reply:
It's true that we use it when closing the contract too, but you better use the `initial_price` to construct the payout function if you want it to be the same one again!